src/usage/UsageToday.tsx

```diff
@@ -119,8 +72,12 @@ const UsageToday: FC = () => {
         >
           <GraphTypeSwitcher
             fromFluxResult={rateLimits}
-            graphInfo={rateLimitGraphInfo}
-            key={rateLimitGraphInfo.title}
+            usageVector={{
+              name: 'Limit Events',
+              fluxKey: '_value',
+              unit: '',
+            }}
```

Comment:
The API doesn't return this is as a source of truth so we're just gonna stick it here